manifests/zzz_profile.yaml

```diff
@@ -41,3 +41,9 @@ Finally, we can set all of that under .Values so the chart behaves without aware
 {{- $a := mustMergeOverwrite $defaults $globals  }}
 {{- end }}
 {{- $b := set $ "Values" (mustMergeOverwrite $defaults $.Values) }}
+
+{{- define "base-labels" -}}
+istio.io/rev: {{ .Values.revision | default "default" | quote }}
+install.operator.istio.io/owning-resource: {{ .Values.ownerName | default "unknown" }}
+release: {{ .Release.Name }}
+{{- end -}}
```

Comment:
Can you 

1. Add the (more current) labels used here to this shared template as well?: https://github.com/istio/istio/blob/master/manifests/charts/ztunnel/templates/_helpers.tpl

2. Remove that ztunnel-local template in favor of this globally-shared one, once that is done.

Reply:
(also cc @ilrudie for :eye: )

Reply:
Thanks!

Reply:
(I left some more detail here on how I'd like to see this pan out in the reply to your comment in https://github.com/istio/istio/issues/52034#issuecomment-2263687886)

Reply:
seems good. the ztunnel-specific template stuff was nothing I'm married to.

would definitely prefer to keep the `app.kubernetes.io/name: ztunnel` and `app.kubernetes.io/part-of: istio` labels around somehow but if they become centralized and folks can can generally rely on their existence everywhere thats great.